test/core/integration/concrete/lockup-base/create-with-timestamps/createWithTimestamps.t.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.22 <0.9.0;
+
+import { IERC721Errors } from "@openzeppelin/contracts/interfaces/draft-IERC6093.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+import { UD60x18, ud } from "@prb/math/src/UD60x18.sol";
+import { Errors } from "src/core/libraries/Errors.sol";
+import { Lockup } from "src/core/types/DataTypes.sol";
+import { Integration_Test } from "./../../../Integration.t.sol";
+
+abstract contract CreateWithTimestamps_Integration_Concrete_Test is Integration_Test {
+    /*//////////////////////////////////////////////////////////////////////////
+                                      HELPERS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @dev Helpers function to assert the common storaged values of a stream between all Lockup models.
+    function assertEqStream(uint256 streamId) internal view {
```

Comment:
we would need to pass the `lockup` contract as well, then

this is why placed it here

Reply:
and why cant we do that?

Reply:
I understand why we cant do because default values are not accessible inside `Assertions`. However, in that case I am in favour of having separate asserts in each file than placing it in the test file. I will merge this  but I am not in favour of it.

Reply:
Marking this as resolved to speed up things with the PR.